experiments/benchmarks/land.jl

```diff
@@ -221,116 +122,34 @@ function setup_prob(t0, tf, Δt; nelements = (101, 15))
-
-    # clm_data is used for g1, vcmax, rooting, and two_stream param maps
-    clm_artifact_path = ClimaLand.Artifacts.clm_data_folder_path(; context)
-
-    # Foliage clumping index data derived from MODIS
-    modis_ci_artifact_path =
-        ClimaLand.Artifacts.modis_ci_data_folder_path(; context)
-
-    # TwoStreamModel parameters
-    Ω = SpaceVaryingInput(
-        joinpath(modis_ci_artifact_path, "He_et_al_2012_1x1.nc"),
-        "ci",
-        surface_space,
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-    χl = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "xl",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-    G_Function = CLMGFunction(χl)
-    α_PAR_leaf = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "rholvis",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-    τ_PAR_leaf = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "taulvis",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-    α_NIR_leaf = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "rholnir",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-    τ_NIR_leaf = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "taulnir",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-
-    # Conductance Model
-    # g1 is read in units of sqrt(kPa) and then converted to sqrt(Pa)
-    g1 = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "medlynslope",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-        file_reader_kwargs = (; preprocess_func = (data) -> data * 10^(3 / 2),),
-    )
-
-    #Photosynthesis model
-    # vcmax is read in units of umol CO2/m^2/s and then converted to mol CO2/m^2/s
-    Vcmax25 = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "vcmx25",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-        file_reader_kwargs = (; preprocess_func = (data) -> data / 1_000_000,),
-    )
-    # photosynthesis mechanism is read as a float, where 1.0 indicates c3 and 0.0 c4
-    is_c3 = SpaceVaryingInput(
-        joinpath(clm_artifact_path, "vegetation_properties_map.nc"),
-        "c3_dominant",
-        surface_space;
-        regridder_type,
-        regridder_kwargs = (; extrapolation_bc,),
-    )
-
     # Plant Hydraulics and general plant parameters
     SAI = FT(0.0) # m2/m2
```

Comment:
scalar variables hang around though

Reply:
You can also set scalars in the same way you are doing now

Reply:
I think we can wait on this. I mainly wanted to get the spatially varying ones in because this was so much code duplicated in many files, and because it makes it easier to ensure our global tests run the same in CI vs long runs vs benchmarks. While the same is true for scalar parameters, I dont want to overengineer anything now since what is spatially varying vs scalar will start to change as we move away from prescribed spatially varying parameters. 

If you feel strongly let me know, otherwise this is ready for squash and merge, after your approval

Reply:
No problem